src/tests.builds

```diff
@@ -11,7 +11,8 @@
   <Import Project="$(ToolsDir)VersionTools.targets" Condition="Exists('$(ToolsDir)VersionTools.targets')" />
 
   <ItemGroup>
-    <Project Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj" Exclude="@(TestProjectExclusions)" />
+    <Project Condition="'$(Performance)'!='true'" Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj" Exclude="@(TestProjectExclusions)" />
```

Comment:
msbuild isn't very smart about how it does file traversals and so it will actually traverse the file system twice here no matter what Performance is set to. In order to help eliminate that double traversal can you please change this to set the search pattern based on Performance. Something like:

```
<PropertyGroup>
  <_ProjectPattern Condition="'$(Performance)'!='true'>.Tests</_ProjectPattern>
  <_ProjectPattern Condition="'$(Performance)'=='true'>.Performance.Tests</_ProjectPattern>
</PropertyGroup>

<ItemGroup>
  <Project Include="$(MSBuildThisFileDirectory)*\tests\**\*$(_ProjectPattern).csproj" Exclude="@(TestProjectExclusions)" />
</ItemGroup>

```